packages/mobile/src/transactions/reducer.ts

```diff
@@ -14,15 +16,17 @@ export interface State {
   // tx feed query watcher. Necessary so we don't re-process
   // txs more than once.
   knownFeedTransactions: KnownFeedTransactionsType
+  recentTxRecipientsCache: NumberToRecipient
 }
 
 export interface KnownFeedTransactionsType {
-  [txHash: string]: boolean
+  [txHash: string]: string | boolean
```

Comment:
is it worth just making these all strings? it looks like boolean true represents it being an exchange - is there any other type represented?

Reply:
We could do this, we're not really using the boolean value for anything atm

Reply:
I believe Exchange and Transfer are the only two types of transactions. We could make them all strings and set something arbitrary for exchanges but leaving it as is also makes it easy to differentiate between the two.

Is there a concern that I may not be considering?

Reply:
Not a huge concern - more making it clear what the existence of a value in this object means. I had to look at the other code to figure out tx -> string with tx.address and escrow -> boolean true

Reply:
Gotcha, will add a comment to make that distinction clear